src/pages/Recipes/index.tsx

```diff
@@ -141,17 +152,19 @@ const Recipes = () => {
                             ))}
                         </ol>
                         <div>
-                            <span>Instructions:</span>
-                            <textarea
-                                value={newRecipe.instructions}
-                                onChange={onNewRecipeInstructionsChange}
-                            />
+                            <label>Instructions:
+                                <textarea
+                                    rows={5}
```

Comment:
Move this number to constants and use it from there like this:

**constants.ts**
`const DEFAULT_TEXTAREA_ROWS = 5`

**Recipes/index.tsx**
`rows={DEFAULT_TEXTAREA_ROWS}`

Reply:
Done